web-portal/store/index.js

```diff
@@ -113,7 +119,19 @@ export const actions = {
         context.commit('UPDATE_USER_DATA', _response.data)
       })
   },
+  LoadAllCalendarEventData: function (context) {
+    context.commit('SET_LOADING_STATUS', true)
 
+    return this.$apiService.get(CURRENT_EVENTS_VERIFIED_PATH)
+      .then((_response) => {
+        context.commit('UPDATE_CALENDAR_EVENTS', _response.data.events)
+        context.commit('SET_LOADING_STATUS', false)
+      })
+      .catch((error) => {
+        console.error(error)
+        context.commit('ui/SHOW_NOTIFICATIONS', { open: true, message: 'Hrrmm... unable to get event data. Please contact us and we will figure out what went wrong.' }, { root: true })
+      })
+  },
```

Comment:
Do we need a second code path for getting the calendar events? Seems like the `LoadAllLocalEventData` path would pull in everything we need -- current + verified events.

Reply:
I made it for future optimization for calendar queries or additional params for the calendar data (like ?embed=Venue used for local events) but no problem with it, I removed extra store props 9a76f5848a479dc3e571d1f062e91c37e6d57c3a